### PR TITLE
RATEST-132: Fix Capture Vitals Test

### PR DESCRIPTION
**Pull request by kdaud**
Ticket ID: https://issues.openmrs.org/browse/RATEST-132

Description:
Fix Capture Vitals Test

### CODE REVIEW

File: ui-tests/src/test/java/org/openmrs/reference/CaptureVitalsTest.java [hunk 1]
```diff
@@ -27,17 +27,12 @@
...
-    @After
-    public void tearDown() throws Exception {
-        deletePatient(patient);
+        new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
     }
 
-    @Ignore
+    @Test
     @Category(BuildTests.class)
-    public void captureVital() throws InterruptedException {
```
Comment (sherrif10) [inline ui-tests/src/test/java/org/openmrs/reference/CaptureVitalsTest.java:40]: is this a typo error, does it break when it remains captureVital

Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/CaptureVitalsTest.java:40]: > is this a typo error, does it break when it remains captureVital

No!. However its our role as QA Engineer to align to the OpenMRS software standards(conventions). Actually its supposed to be _captureVitalsTest()_

Comment (sherrif10) [reply ui-tests/src/test/java/org/openmrs/reference/CaptureVitalsTest.java:40]: Thats the exact answer i wanted from you , thanks and well done

File: ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java [hunk 2]
```diff
@@ -31,7 +31,7 @@
     private static final By CONFIRM = By.cssSelector("#quick-visit-creation-dialog .confirm");
     private static final By STARTED_AT = By.className("active-visit-started-at-message");
     private static final By VISIT_NOTE = By.id("referenceapplication.realTime.simpleVisitNote");
```
Comment (kaweesi) [inline ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java:33]: this should also be affected

File: ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java [hunk 3]
```diff
@@ -31,7 +31,7 @@
...
-    private static final By PATIENT_CAPTURE_VITALS = By.id("referenceapplication.realTime.vitals");
+    private static final By PATIENT_CAPTURE_VITALS = By.linkText("Capture Vitals");
```
Comment (kaweesi) [inline ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java:34]: replace with an element selector
`By.cssSelector("a[id='referenceapplication.realTime.vitals']")`

File: ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java (hunk 2 above)
Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java:33]: > this should also be affected

_Boolean value=true_

File: ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java (hunk 3 above)
Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java:34]: Thanks @kaweesi . Caught this in the updated commit

Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java]: After simulating the test, the element PATIENT_CAPTURE_VITALS is retrievable by the existing attribute.

Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java]: After simulating the test, the element PATIENT_CAPTURE_VITALS is retrievable by the existing attribute.

Comment (k-joseph) [reply ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java]: what did u finally do with this?

Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java]: The element is retrievable by the existing set attribute. Changing to _cssSelector_ could be alternative to retrieving the element. @k-joseph should I go for _cssSelector_

Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java:34]: This is @kdaud  once again! 
After simulating the behaviour of the element (PATIENT_CAPTURE_VITALS) a couple of times, I observed that the element fails to be retrieved by selenium API with the existing attribute. Took time to use different look ups via is (i_d,cssSelector, xpath, and others_) and realised the element only fails with chrome headless =true. Simulated a use case when retrieving the element via By.cssSelector("a[id='referenceapplication.realTime.vitals']")  and similar failure occurs as well. The best look up for this element is via _linkText()_ which passes on both chrome and firefox instances. cc: @k-joseph

**Pull request merged**